code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -42,6 +41,8 @@
 	var/use_sound = /singleton/sound_category/rustle_sound	//sound played when used. null for no sound.
 	var/list/starts_with // for pre-filled items
 	var/empty_delay = 0 SECOND // time it takes to empty bag. this is multiplies by number of objects stored
+	///Boolean, whether or not we should have those cute little animations
```

Comment:
What animations? This should explain what is the "animation" in this context. That being the item taking-out or putting-in animation (?).